client/transaction-pool/src/graph/pool.rs

```diff
@@ -100,7 +100,7 @@ pub trait ChainApi: Send + Sync {
 }
 
 /// Pool configuration options.
-#[derive(Debug, Clone)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
```suggestion
#[derive(Debug, Clone)]
```

Reply:
Why is using `Clone` for effectively `Copy` types preferred?

Reply:
Almost every type is constructed of copyable types, which doesn't mean that the resulting type should be copyable.